src/style.css

```diff
@@ -1,3 +1,83 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav_item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  text-decoration: none;
+  display: flex;
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+  white-space: nowrap;
+  font-weight: 500;
+  height: 60px;
+  align-items: center;
+  justify-content: center;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+/* [data-qa='hover'] {
+  color: #00acdc;
+} */
+
+:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  height: 4px;
+  width: 100%;
+  content: '';
+  background-color: #00acdc;
```

Comment:
CSS Variable is used for a blue color
<img width="857" alt="image" src="https://github.com/user-attachments/assets/ec3d0887-ab88-44da-b11c-04ff41c87025">